builds/android/res/values-de/strings.xml

```diff
@@ -31,7 +32,49 @@
 	<string name="not_valid_game">$PATH ist kein gültiges Spiel</string>
 	<string name="accessing_configuration_failed">Konfiguration von $PATH konnte nicht gelesen werden.</string>
 	<string name="select_game_region">Spielregion auswählen</string>
+	<string name="change_the_layout">Layout ändern</string>
+	<string name="choose_layout">Layout auswählen</string>
 	<string name="unknown_region">Unbekannte Region</string>
-	<string name="region_modification_success">Region in $NAME geändert</string>
 	<string name="region_modification_failed">Ändern der Region fehlgeschlagen</string>
+	<string name="refresh">Aktualisieren</string>
+	<string name="change_default_mapping">Standard-Tastenlayout ändern</string>
+	<string name="how_to_use_easy_rpg">Wie verwende ich EasyRPG</string>
+	<string name="how_to_use_easy_rpg_explanation">
+Installation:\n
+\t1. Platziere die Spiele in /easyrpg/games im Hauptverzeichnis deines Geräts.\n
+\t2. Platziere das RTP in /easyrpg/rtp/2000 für das RPG Maker 2000 RTP bzw. in /easyrpg/rtp/2003 für das RPG Maker 2003 RTP.\n
+\n
+Wenn Spiele nicht erkannt werden und dein Gerät hat mehr als einen Speicher (z.B. MicroSD) zur Verfügung, versuche den easyrpg-Ordner auf den anderen Speicher zu kopieren und drücke "Aktualisieren".\n
```

Comment:
I think this one is better: `Wenn Spiele nicht erkannt werden und dein Gerät mehr als ein Speichermedium (z.B. MicroSD) zur Verfügung hat, versuche den easyrpg Ordner auf das andere Speichermedium zu kopieren. Tippe danach auf "Aktualisieren".`